vars/valgrindReportPublish.groovy

```diff
@@ -30,17 +30,17 @@ def call(Map config = [:]) {
     // Need to have only one Valgrind publish stage
     println "No valgrind_stashes passed!   Running older code!"
   }
-  
-  sh label: 'Remove *.memcheck.xml files in this workspace',
-     script: '''rm -rf *.memcheck.xml'''
+
+  fileOperations([fileDeleteOperation(includes: '*.memcheck.xml',
+                                      excludes: '')])
```

Comment:
Is the empty excludes necessary?  Seems like that would be the default value.

Your experiments with trying to delete a pattern revealed that it didn't cause any errors or exceptions?

Reply:
You're right, I think it's not necessary. I sent a patch removing it. 

And yes, the delete with fileOperations did not cause any problems.

Reply:
@marcelarosalesj Please don't resolve conversations you didn't start.  It's up to the person who made the initial comment to resolve the conversation when they are satisfied that their issue was resolved.